src/amber/router/context.cr

```diff
@@ -15,7 +15,6 @@ class HTTP::Server::Context
   def initialize(@request : HTTP::Request, @response : HTTP::Server::Response)
     @router = Amber::Router::Router.instance
     parse_params
```

Comment:
If we're already parsing params here, what do we gain via removing upgrade_request_method here? That takes no time at all.